wfl/Controllers/MenuSelectionsController.cs

```diff
@@ -32,6 +32,16 @@ public class MenuSelectionsController : ApiController
             return Ok(order.MenuSelections[index]);
         }
 
+        [Route("{index}")]
+        [HttpDelete]
+        public async Task<IHttpActionResult> DeleteByIdAsync([FromUri] string foodOrderId, [FromUri] int index)
```

Comment:
see my comment in the submit menu selection PR (where the MenuSelection model is defined) but I think this property `index` is problematic and needs to be replaced with an id.